utilities/hydrofabric_test/README.md

```diff
@@ -0,0 +1,67 @@
+# Testing ngen and t-route on a given hydrofabric
+
+Create and activate a virtual environment
+```sh
+python -m venv venv
+source venv/bin/activate
+```
+
+Update pip and install required modules
+```sh
+pip install -U pip
+pip install -r requirements.txt
+```
+
+## Make sure t-route modules are installed in this venv
+
+If you have already built/installed t-route in another environment, you can reuse the packages and install them in this test environment
+
+```sh
+pip install t-route/src/ngen_routing/
```

Comment:
In line with some of @aaraney 's other comments, this list of pip installs is wrong for t-route master branch.